sdk-python/littlehorse/worker.py

```diff
@@ -33,7 +33,7 @@
 
 REPORT_TASK_DEFAULT_RETRIES = 5
 HEARTBEAT_DEFAULT_INTERVAL = 5
-HEALTH_TIMEOUT = 60000
+GRPC_UNARY_CALL_TIMEOUT = 1
```

Comment:
@coltmcnealy-lh this is important, is 1sec OK?

Reply:
Way too small...at least 20 seconds I think. In certain cases, if the LH Server is in a different region from the task worker, the round-trip latency might be close to 500 milliseconds without any dropped packets. Where is this used?

Reply:
sure